qa/tasks/cephfs/test_forward_scrub.py

```diff
@@ -41,7 +42,7 @@ def _read_str_xattr(self, pool, obj, attr):
 
     def _get_paths_to_ino(self):
         inos = {}
-        p = self.mount_a.run_shell(["find", "./"])
+        p = self.mount_a.run_shell(args=["find", "./"], stdout=StringIO())
```

Comment:
this is useless, just use remote.sh, like this
```
paths = self.mount_a.client_remote.sh("find ./", cwd=self.mount_a.mountpoint).strip().split()
```

Reply:
I have removed the run_shell() in vstart_runner.py and reuse the one from the parent class.

Reply:
Rishabh have fix the partially fixed it in https://github.com/ceph/ceph/pull/34595, and for the py3 compatible is fixing in https://github.com/ceph/ceph/pull/34781.